src/simulator/perftest_report_dstat.py

```diff
@@ -22,8 +22,8 @@ def analyze_dstat(run_dir, attributes):
         agent_id = csv_filename[:csv_filename.index("_")]
         csv_path = f"{run_dir}/{csv_filename}"
         info(f"\tLoading {csv_path}")
-        df = pd.read_csv(csv_path, skiprows=6)
-        multiple_by(df, 1000, 'used', 'free', 'buff', 'cach')
+        df = pd.read_csv(csv_path, skiprows=5)
```

Comment:
The original change is correct, if you are using 

```
    #default AWS AMI
    ami: <ami>
    user: ec2-user
```

I wonder should we have a note or something, maybe a comment in the inventory.yaml.. as it does blow things up if you use ec2-user

Reply:
https://github.com/hazelcast/hazelcast-simulator/pull/2248